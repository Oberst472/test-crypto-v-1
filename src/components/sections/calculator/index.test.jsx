import {render, screen} from '@testing-library/react'

 import Calculator from './index'


 describe('CALCULATOR TEST', () => {
     test('Section Calculator renders', () => {
         render(<Calculator/>)
         expect(screen.getByText(/Select your Amount/i)).toBeInTheDocument()
     })
 })
