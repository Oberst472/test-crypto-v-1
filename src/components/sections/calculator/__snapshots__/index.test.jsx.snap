// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CALCULATOR TEST fetch info about currencies 1`] = `
<button
  class="ui-btn section-calculator__btn"
>
  Buy now
</button>
`;
